### ISSUE TITLE
Make tooltips preserve the menu option's colours

### ISSUE THREAD

**Issue by Enriath** [ryanxie0/lingering-click-tooltips#1]
Currently, the text of the tooltips is set to be entirely white. It would be great if it actually used the colours present in the menu option

**Comment by ryanxie0** [comment]
This is a feature I wanted to have working, but hadn't considered as top priority.

Truth be told, I borrowed code from another plugin for generating renderable components and for some reason there was no way to apply color to the text in the components provided. In short, this would require a little more research on my end and a possible rework of the renderable components as a whole.

Considering there's not even opacity for when using the user-selected custom background color, I would consider doing a renderable rework in the near future.

**Comment by Enriath** [comment]
Should be pretty simple tbh; you're purposely stripping the colour tags from the text at the start of
https://github.com/ryanxie0/lingering-click-tooltips/blob/master/src/main/java/ryanxie0/runelite/plugin/lingeringclicktooltips/util/LingeringClickTooltipsUtil.java#L53

Speaking of that function, you can also build the text with a much simpler call
```java
option + (Strings.isNullOrEmpty(target) ? "" : " " + target)
```

**Comment by ryanxie0** [comment]
Yeah, I had to strip the tags for building the actual text of the tooltips. I figured that users don't want to see weird tags on their tooltips. But I actually never put 2 + 2 together and you've made me realize that those tags are the actual color of the text, which is useful to know.

I'd need to add the condition for where they're equal to your statement. But other than that, this is indeed a cleaner way to build the text.
